### PR TITLE
chore: adds settings file to apdex tests

### PR DESCRIPTION
**Pull request by latin-panda**
# Description
Adds settings provider to run tests based on a settings file. 

1. Create a settings file: 
```
{
  "instanceURL": "https://<add-the-rest>",
  "appium": {
    "platformVersion": "<add-android-version>",
    "deviceName": "<add-device-name>"
  },
  "users": [
    {
      "type": "offline",
      "role": "chw",
      "username": "<add-username>",
      "password": "<add-password>"
    }
  ]
}
```

2. Set the environment variable  `APDEX_TEST_SETTINGS` with the path of your settings file. 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:apdex-script-fill-forms/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:apdex-script-fill-forms/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:apdex-script-fill-forms/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@lorerod @ralfudx, this is the first part of the settings file, which is the minimum required to make it run. 
I added the minimum documentation to run this with the settings file. 

@ralfudx, please add more documentation in your branch [apdex-automation-tests](https://github.com/medic/cht-core/tree/apdex-automation-tests) to make it easy for others. 

I'll send the navigation to the contact page tomorrow.

File: tests/performance/apdex-score/README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+## Setup
+
+1. Enable the developer mode in your phone and enable the USB Debugger mode.
+2. Connect the phone to the computer
+3. Create a settings file:
+```
+{
+  "instanceURL": "<instance_url>",
+  "appium": {
```
Comment (latin-panda) [inline tests/performance/apdex-score/README.md:11]: Make this array

Comment (latin-panda) [reply tests/performance/apdex-score/README.md]: Done

File: tests/performance/apdex-score/wdio.conf.js [hunk 2]
```diff
@@ -169,8 +171,17 @@ exports.config = {
    * @param {object} config wdio configuration object
    * @param {Array.<Object>} capabilities list of capabilities details
    */
-  // onPrepare: function (config, capabilities) {
-  // },
+  onPrepare: function (config, capabilities) {
+    try {
+      const settingsCapabilities = loadSettings().getCapabilitiesSettings()[0];
+      const appium = capabilities[0];
```
Comment (ralfudx) [inline tests/performance/apdex-score/wdio.conf.js:177]: qq - I'm assuming we'd need to modify this for multiple capability entries?

**Review by ralfudx — APPROVED** [review]
LGTM

**Review by ralfudx — APPROVED** [review]
Still looks good

**Pull request merged**